### DISCUSSION TITLE
Add "Album to Library" on Album page

### DISCUSSION THREAD

**Discussion by JustJJ007** [z-huang/music#764] [Ideas]
I noticed there isn't an "Add to library" button on album pages, perhaps is it possible to add that?

**Comment by TheTrashCrafter** [comment]
![Screenshot_20230707-192131](https://github.com/z-huang/InnerTune/assets/107064092/18ae0615-5dd4-4755-a206-e41d88c23914)

There is one.

Edit: Only on 0.5.0+ though
